package.json

```diff
@@ -76,7 +76,6 @@
     "less": "^4.0.0",
     "less-loader": "^8.0.0",
     "lint-staged": "^13.2.1",
-    "loader-utils": "^2.0.3",
```

Comment:
Should we bump it to v3 as well here?

Reply:
We don't need it, `loader-utils` should be deprecated, we need to implement only one function on `webpack` side - `interpolateName`, we already migreate from it in loaders and plugins, time to finish migration